packages/core/src/App/Containers/Layout/header/default-header.jsx

```diff
@@ -91,6 +93,32 @@ const DefaultHeader = ({
             }
             return true;
         });
+
+    const Divider = () => {
+        return <div className='header__menu--separator' />;
+    };
```

Comment:
Can you reuse `Divider` component that we already have?

Reply:
Some dividers require different css

Reply:
What do you mean? you can pass a `className` as a prop to the component